modules/pvc-autoscale/data/pvc_autoscale_notebook.json

```diff
@@ -0,0 +1,162 @@
+{
+  "cells": [
+    {
+      "content": "# Setup\nInstall required packages and configure pods for testing.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "## Setup Kubernetes\nSetup kubernetes objects for the test.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app =~ \"shoreline\" | limit=1 | `kubectl apply -f  \"/tmp/pvc_test.yaml\"`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "If you ran into permission problem, run the apply command (kubectl apply -f \"/tmp/pvc_test.yaml\") with a user having sufficient permission. Content of the k8s manifest file:",
```

Comment:
Similar to JVM, maybe remove the `/tmp` and indicate that the manifest comes from running the command below.